ArticleShapeExtractor/extensions/globals.js

```diff
@@ -14,4 +14,23 @@ globalThis.alert = function(msg, caption) {
     theDialog.show();
     theDialog.destroy();
     return;
+}
+
+/**
+ * Compare two versions.
+ * @param {String} versionLhs Left Hand Side.
+ * @param {String} versionRhs Right Hand Side.
+ * @returns {Number} Zero when equal, or -1 when versionLhs is smaller or 1 when bigger.
+ */
+globalThis.versionCompare = function(versionLhs, versionRhs) {
```

Comment:
Is it really necessary to add this to global? Why can't it be an util that exports a function?